cmd/kops/create_cluster.go

```diff
@@ -500,6 +504,10 @@ func RunCreateCluster(ctx context.Context, f *util.Factory, out io.Writer, c *Cr
 		cluster.Spec.NetworkCIDR = c.NetworkCIDR
 	}
 
+	if c.IPv6 {
+		cluster.Spec.NonMasqueradeCIDR = "fd00:10:96::/56"
```

Comment:
Why this address? The `10:96` is not random as the spec requires.

Reply:
If we are going to automatically assign the `NonMasquereadeCIDR` it should come from the VPC's IPv6 allocation. We should not automatically assign a ULA, especially if it isn't U.

Reply:
For IPv4 we were using a fixed IP, but don't mind making it random.
What would be the benefit of using a block from the VPC IPv6 allocation?
Should I just remove this and discuss separately?
Is this a good case for making `NonMasquereadeCIDR` user configurable on `create cluster`?

Reply:
Please remove this and discuss separately.

A benefit of using a block from the VPC IPv6 allocation is that one could then choose to make pods directly accessible from outside the cluster. Also, one wouldn't have to NAT the IPv6 outbound from pods, so the IPv6 destinations would be logging the IPv6 addresses of the pods.

I believe `NonMasqueradeCIDR` could be set using SpecOverride. Possible future work would be to default it to "/64#0" (and make that work).

Reply:
FYI, making "/64#0" work for `NonMasqueradeCIDR` depends on work in my #9229 track. In that track I plan to refactor the writing of the completed cluster spec into a `ManagedFile` task. Once that is done, `PopulateClusterSpec` would need to be turned into a `Task` and made dependent on `VPCAmazonIPv6CIDRBlock` for managed VPCs.